packages/models/src/index.ts

```diff
@@ -73,6 +75,8 @@ export * from "./gen/v1/agreement/events.js";
 export * from "./gen/v1/agreement/state.js";
 export * from "./gen/v1/tenant/tenant.js";
 export * from "./gen/v1/tenant/events.js";
+export * from "./gen/v2/tenant/tenant.js";
+export * from "./gen/v2/tenant/events.js";
```

Comment:
these are already [here](https://github.com/pagopa/interop-be-monorepo/pull/435/files#diff-7794d0c612323dc86c27b6bd9058d68c031894b178b93ec2dd4b88ee57bae541R100-R101) 

hint: keep them sorted, this way it's easier to spot duplicates
```suggestion
```